shell/platform/darwin/ios/BUILD.gn

```diff
@@ -153,6 +143,142 @@ shared_library("create_flutter_framework_dylib") {
   ]
 }
 
+static_library("ocmock") {
+  configs -= [ "//build/config/compiler:chromium_code" ]
+  cflags = [
+    "-fvisibility=default",
+    "-F/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform//Developer/Library/Frameworks/",
```

Comment:
1. What if Xcode is at a different path?
2. What about the non-simulator frameworks?

Reply:
This is still a draft PR =).  I can address where xcode is located.  I haven't been able to get rid of the hardcoded iPhoneSimulator path.  The problem is that GN special cases all libs that are "X.framework" but it doesn't support XCTest.framework.

Since this is intended for the simulator it isn't the end of the world if I can't get it removed.

Reply:
> This is still a draft PR =).

Sorry, got excited!

Reply:
done